owncloudApp/src/main/java/com/owncloud/android/extensions/ActivityExt.kt

```diff
@@ -242,8 +242,8 @@ fun Activity.openFileWithIntent(intentForSavedMimeType: Intent, intentForGuessed
 fun AppCompatActivity.sendFile(file: File?) {
     if (file != null) {
         val sendIntent: Intent = makeIntent(file, this)
-        // Show dialog, without the own app
-        val packagesToExclude = arrayOf<String>(this.packageName)
+
+        val packagesToExclude = arrayOf<String>()
```

Comment:
What is the sense of creating a variable for an empty array? Send `arrayOf<String>()` instead of creating a variable for that empty array

Reply:
done